### PR TITLE
Update LocalDevice.clientId

### PR DESCRIPTION
**Pull request by ricardopereira**
#877

### CODE REVIEW

File: Source/ARTPushActivationStateMachine.m [hunk 1]
```diff
@@ -226,6 +226,7 @@ - (void)deviceUpdateRegistration:(ARTErrorInfo *)error {
     NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[[NSURL URLWithString:@"/push/deviceRegistrations"] URLByAppendingPathComponent:local.id]];
     request.HTTPMethod = @"PATCH";
     request.HTTPBody = [[_rest defaultEncoder] encode:@{
+        @"clientId": local.clientId,
```
Comment (ricardopereira) [inline Source/ARTPushActivationStateMachine.m]: @paddybyers Is this necessary for the new release?

Comment (ricardopereira) [reply Source/ARTPushActivationStateMachine.m]: Removed that line because ClientId should be immutable after a successful device registration.

**Comment by ricardopereira** [comment]
@paddybyers updated for the wildcard case.

**Comment by paddybyers** [comment]
> updated for the wildcard case

LGTM, thanks - lets see what happens with the tests

**Pull request merged**